.env.deploy

```
@@ -3,4 +3,5 @@ REACT_APP_REQUEST_BROKER_BASEURL=${REQUEST_BROKER_BASEURL}
 REACT_APP_LOCALSTORAGE_KEY=DIMESMyList
 REACT_APP_MINIMAP_KEY=DIMESMinimapIntro
 REACT_APP_S3_BASEURL=${AWS_BUCKET_BASEURL}
-REACT_APP_CAPTCHA_SITE_KEY=${CAPTCHA_SITE_KEY}
\ No newline at end of file
+REACT_APP_CAPTCHA_SITE_KEY=${CAPTCHA_SITE_KEY}
+REACT_APP_ENABLE_READING_ROOM_SELECT=false
```

```suggestion
REACT_APP_ENABLE_READING_ROOM_SELECT=${ENABLE_READING_ROOM_SELECT}
```
Let's use an environment variable here so we can not implement.